commitlint/helpers.ts

```diff
@@ -22,13 +22,7 @@ export abstract class Helpers {
     ): string {
         if (potentialString === null || potentialString === undefined) {
             // otherwise, String(null) might give us the stupid string "null"
-            throw new Error(
-                "Unexpected " +
-                    paramName +
-                    "===null or " +
-                    paramName +
-                    "===undefined happened"
-            );
+            return "";
```

Comment:
@realmarv this needs to be fixed; already discussed in other PR

Reply:
Done